bin/plugin/commands/performance.js

```diff
@@ -147,15 +133,12 @@ function curateResults( testSuite, results ) {
 		testSuite === 'front-end-classic-theme' ||
 		testSuite === 'front-end-block-theme'
 	) {
+		const timeToFirstByte = median( results.timeToFirstByte );
+		const largestContentfulPaint = median( results.largestContentfulPaint );
 		return {
-			timeToFirstByteMedian: median( results.timeToFirstByte ),
-			timeToFirstByteP75: percentile75( results.timeToFirstByte ),
-			largestContentfulPaintMedian: median(
-				results.largestContentfulPaint
-			),
-			largestContentfulPaintP75: percentile75(
-				results.largestContentfulPaint
-			),
+			timeToFirstByte,
```

Comment:
This is the gist, the rest are docs changes:

- rename `timeToFirstByteMedian` to `timeToFirstByte`
- remove `timeToFirstByteP75`
- rename `largestContentfulPaintMedian` to `largestContentfulPaint`
- remove `largestContentfulPaintP75`
- add `lcMinusTtfb`

I presume this is going to need to `codevitals.run` config changes to gather the existing metrics we report for the updated fields. If renaming is problematic for codevitals, I can revert that and leave the `*Median` fields while still removing the `*P75` ones.

Can we also should the new one `lcpMinetTtfb` as `LCP - TTFB (classic or block)`, @youknowriad ?